### PR TITLE
Fix Docs

### PR DESCRIPTION
**Pull request by kulothunganug**
### Description of Changes
* Fix typo (a and an)
* Removed `get_color_from_hex`
> When handling with `ColorProperty` we can directly give a hex color or rgba it would recognize automatically.
> ex: "FFFFFF", "#FFFFFF", (1, 1, 1, 1)

#### See files by commit to get a clear understand

### CODE REVIEW

File: kivymd/theming.py [hunk 1]
```diff
@@ -40,7 +40,6 @@
 
 from kivymd import images_path
 from kivymd.color_definitions import colors, hue, palette
-from kivymd.font_definitions import theme_font_styles  # NOQA: F401
```
Comment (HeaTTheatR) [inline kivymd/theming.py:43]: It cannot be deleted (Flake8).

**Review by HeaTTheatR — CHANGES_REQUESTED** [review]

File: kivymd/uix/backdrop.py [hunk 2]
```diff
@@ -218,7 +218,7 @@ class MDBackdrop(ThemableBehavior, FloatLayout):
     """
     Padding for contents of the front layer.
 
-    :attr:`padding` is an :class:`~kivy.properties.ListProperty`
+    :attr:`padding` is a :class:`~kivy.properties.ListProperty`
```
Comment (HeaTTheatR) [inline kivymd/uix/backdrop.py:221]: https://github.com/kivy/kivy/blob/master/kivy/uix/codeinput.py#L70

**Review by HeaTTheatR — CHANGES_REQUESTED** [review]

File: kivymd/uix/dialog.py [hunk 3]
```diff
@@ -181,7 +181,11 @@ def show_alert_dialog(self):
 
 
 class BaseDialog(ThemableBehavior, ModalView):
-    radius = ListProperty([7, 7, 7, 7])
+    radius = ListProperty(
+        [
+            dp(7),
+        ]
+    )
```
Comment (HeaTTheatR) [inline kivymd/uix/dialog.py:188]: This is an error - the radius is intended to change the value of all angles, not just one. The four-element list means that you can set different values for any angles.

**Review by HeaTTheatR — CHANGES_REQUESTED** [review]

File: kivymd/theming.py [hunk 4]
```diff
@@ -731,7 +731,7 @@ def _determine_device_orientation(self, _, window_size):
     """
     Device orientation.
 
-    :attr:`device_orientation` is an :class:`~kivy.properties.StringProperty`.
+    :attr:`device_orientation` is a :class:`~kivy.properties.StringProperty`.
```
Comment (HeaTTheatR) [inline kivymd/theming.py:734]: https://github.com/kivy/kivy/blob/master/kivy/uix/codeinput.py#L70

**Review by HeaTTheatR — CHANGES_REQUESTED** [review]

File: kivymd/uix/backdrop.py (hunk 2 above)
Comment (kulothunganug) [reply kivymd/uix/backdrop.py:221]: Sorry I can't understand

File: kivymd/theming.py (hunk 1 above)
Comment (kulothunganug) [reply kivymd/theming.py:43]: Does `theme_text_color` used in any other files?
like `from kivymd.theming import theme_font_styles`

File: kivymd/theming.py (hunk 4 above)
Comment (kulothunganug) [reply kivymd/theming.py:734]: Sorry I can't understand

File: kivymd/theming.py (hunk 1 above)
Comment (HeaTTheatR) [reply kivymd/theming.py:43]: If after the import the `# NOQA: F401` tag is set, then it is used and this import cannot be deleted.

File: kivymd/uix/dialog.py (hunk 3 above)
Comment (kulothunganug) [reply kivymd/uix/dialog.py:188]: https://github.com/kivymd/KivyMD/blob/93d6823b3eb6cbd6574f81852536cc548846acad/kivymd/uix/menu.py#L905
https://github.com/kivymd/KivyMD/blob/93d6823b3eb6cbd6574f81852536cc548846acad/kivymd/uix/tooltip.py#L172

before this [commit](https://github.com/kivymd/KivyMD/commit/69680954753c9f2e7cc57504bb918940ea9af5a8#diff-d808ef41d27ee485ae6d3f04e2cb96c5d3774c2696ffbc3299d33b15e7cd3ab3) in mobile the edges are sharp
after the edges are looks same in all platforms

**Pull request closed without merging**